sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetReadSupport.scala

```diff
@@ -140,7 +140,7 @@ object ParquetReadSupport extends Logging {
         "Spark read schema expects field Ids, " +
           "but Parquet file schema doesn't contain any field Ids.\n" +
         "Please remove the field ids from Spark schema or ignore missing ids by " +
-          "setting `spark.sql.parquet.fieldId.ignoreMissing = true`\n" +
+          "setting `spark.sql.parquet.fieldId.read.ignoreMissing = true`\n" +
```

Comment:
To prevent this kind of issue, we prefer `SQLConf.IGNORE_MISSING_PARQUET_FIELD_ID.key` style. Shall we revise?